portal/server/routes/admin/users.js

```diff
@@ -53,11 +53,7 @@ router.get('/users/create', async (req, res) => {
 
 // Admin - user create
 router.post('/users/create', async (req, res) => {
-  const { firstname, surname, roles, bank } = req.body;
-
-  if (!(firstname && surname && roles && bank)) {
-    return res.redirect('/admin/users/create');
-  }
+  const { bank } = req.body;
```

Comment:
Is this required?

Reply:
No, as the form picks validation errors and displays if fields are left empty.